infra/cmake/packages/JsoncppConfig.cmake

```diff
@@ -0,0 +1,24 @@
+function(_Jsoncpp_import)
+  nnas_find_package(JsoncppSource QUIET)
+
+  if(NOT JsoncppSource_FOUND)
+    set(Jsoncpp_FOUND FALSE PARENT_SCOPE)
+    return()
+  endif(NOT JsoncppSource_FOUND)
+
+  nnas_include(ExternalBuildTools)
+  ExternalBuild_CMake(CMAKE_DIR   ${JsoncppSource_DIR}
+                      BUILD_DIR   ${CMAKE_BINARY_DIR}/externals/JSONCPP/build
+                      INSTALL_DIR ${EXT_OVERLAY_DIR}
+                      IDENTIFIER  "1.9.5"
+                      PKG_NAME    "JSONCPP"
+                      EXTRA_OPTS "-DBUILD_STATIC_LIBS=ON"
+                                 "-DBUILD_SHARED_LIBS=OFF")
```

Comment:
This is for using static library

Reply:
Is there any reason not to use so binary ? FYI, tizen's jsoncpp-devel package provides so files.

Reply:
It is for ease of release. I think that using system-installed libraries (ex: jsoncpp-devel in tizen) will make release difficult.

Reply:
Ah static link is better for offline tool. Lets discuss later about tizen case